PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/run/SparkBatchRemoteRunState.kt

```diff
@@ -121,8 +121,6 @@ open class SparkBatchRemoteRunState(private val sparkSubmitModel: SparkSubmitMod
                     },
                     { onComplete(it) })
 
-            programRunner.onProcessStarted(null, executionResult)
```

Comment:
No replacement here?

Reply:
`programRunner.onProcessStarted` is not used anymore,  please check the method definiton below. There is no code reference for this method except our code here.
```
  /**
   * @deprecated Not used by platform.
   */
  @SuppressWarnings("unused")
  @Deprecated
  default void onProcessStarted(RunnerSettings settings, ExecutionResult executionResult) {
  }

```